app/actions/multiSrp/index.test.ts

```diff
@@ -0,0 +1,94 @@
+import { KeyringTypes } from '@metamask/keyring-controller';
+import ExtendedKeyringTypes from '../../constants/keyringTypes';
+import Engine from '../../core/Engine';
+import {
+  importNewSecretRecoveryPhrase,
+  createNewSecretRecoveryPhrase,
+} from './';
+import { wordlist } from '@metamask/scure-bip39/dist/wordlists/english';
+
+jest.mock('../../util/Logger');
+
+const mockSetSelectedAddress = jest.fn();
+const mockAddNewKeyring = jest.fn();
+const mockGetKeyringsByType = jest.fn();
+const mockWithKeyring = jest.fn();
+const mockGetAccounts = jest.fn();
+jest.mock('../../core/Engine', () => ({
+  context: {
+    KeyringController: {
+      addNewKeyring: (keyringType: ExtendedKeyringTypes, args: unknown) =>
+        mockAddNewKeyring(keyringType, args),
+      getKeyringsByType: () => mockGetKeyringsByType(),
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
+      withKeyring: (args: any) => mockWithKeyring(args),
+      getAccounts: () => mockGetAccounts(),
+    },
+  },
+  setSelectedAddress: (address: string) => mockSetSelectedAddress(address),
+}));
+
+jest.mocked(Engine);
+
+const testAddress = '0x123';
+const testMnemonic =
+  'verb middle giant soon wage common wide tool gentle garlic issue nut retreat until album recall expire bronze bundle live accident expect dry cook';
+
+describe('MultiSRP Actions', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+  });
+
+  describe('importNewSecretRecoveryPhrase', () => {
+    it('should import new SRP successfully', async () => {
```

Comment:
S: Follow [unit test naming guidelines](https://github.com/MetaMask/contributor-docs/blob/main/docs/testing/unit-testing.md)
```suggestion
    it('imports new SRP', async () => {
```